make/Makefile.dev

```diff
@@ -0,0 +1,60 @@
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=git-service
+SHORT_COMMIT=$(shell git rev-parse --short HEAD)
+ifneq ($(GITUNTRACKEDCHANGES),)
+SHORT_COMMIT := $(SHORT_COMMIT)-dirty
+endif
+
+TIMESTAMP:=$(shell date +%s)
+TAG?=$(SHORT_COMMIT)-$(TIMESTAMP)
+
+DEPLOY_DIR:=deploy
+
+.PHONY: create-resources
+create-resources:
+	@echo "Logging using system:admin..."
+	@oc login -u system:admin
+	@echo "Creating sub resources..."
+	@echo "Creating CRDs..."
+	@oc create -f https://raw.githubusercontent.com/redhat-developer/devopsconsole-operator/master/deploy/crds/devopsconsole_v1alpha1_gitsource_crd.yaml
+	@echo "Creating Namespace"
+	@oc create -f $(DEPLOY_DIR)/namespace.yaml
+
+.PHONY: create-cr
+create-cr:
+	@echo "Creating Custom Resource..."
+
+.PHONY: build-image
+build-image:
+	docker build -t $(DOCKER_REPO)/$(IMAGE_NAME):$(TAG) -f Dockerfile.dev .
+	docker tag $(DOCKER_REPO)/$(IMAGE_NAME):$(TAG) $(DOCKER_REPO)/$(IMAGE_NAME):test
+
+.PHONY: deploy-apiserver-only
+deploy-apiserver-only:
+	@echo "Creating API Server..."
+
+.PHONY: clean-all
+clean-all:  clean-apiserver clean-resources
+
+.PHONY: clean-apiserver
+clean-apiserver:
+	@echo "Deleting API Server..."
```

Comment:
same here

Reply:
This is a placeholder I would like to keep. We will need to deploy and cleanup the server anyway. But I've renamed it to gitserver instead of apiserver in https://github.com/redhat-developer/git-service/pull/2/commits/26b2bf8d54e7712a2986c39b09a55126a21cffa6

Reply:
ok, fine